Automattic/wp-calypso
# Migrate subscribers: Calypso UI

Related to the Hackathon project, see: Pf4ekF-3L-p2 

## Proposed Changes

Adds functionality to move migration from the older Atlas system to Calypso

## Testing Instructions



* Go to Users > Subscribers
* Click Migrate subscribers from another WordPress.com site
* Test the UI, make sure you get a confirmation.

![CleanShot 2023-11-16 at 16 15 51@2x](https://github.com/Automattic/wp-calypso/assets/533/985f64be-44aa-49f9-bc30-725dfaf5ce7d)

![CleanShot 2023-11-16 at 16 16 12@2x](https://github.com/Automattic/wp-calypso/assets/533/b985483c-c6dd-4cde-b637-492ab7f582c9)


## Pre-merge Checklist
